app/js/components/mediaPlayer.js

```diff
@@ -5,13 +5,15 @@ var Actions           = require('../actions/actionCreators')
 var CurrentTrackStore = require('../stores/currentTrackStore')
 
 var time              = require('../utils/time')
+var github            = require('../utils/github')
 var classNames        = require('classnames')
 var _                 = require('lodash')
 
-var remote            = window.require('remote');
-var Menu              = remote.require('menu');
-var MenuItem          = remote.require('menu-item');
-var pjson             = remote.require('./package.json');
+var remote            = window.require('remote')
+var Menu              = remote.require('menu')
+var MenuItem          = remote.require('menu-item')
+var pjson             = remote.require('./package.json')
```

Comment:
We actually have a `config` module that takes care of reading values from the `package.json`, to maintain consistency I would recommend using that one.
https://github.com/gillesdemey/Cumulus/blob/master/lib/config.js#L22